packages/babel-cli/test/fixtures/babel/--copy-files --include-dotfiles with ignore/stdout.txt

```diff
@@ -1 +1,2 @@
 src/index.js -> lib/index.js
+Successfully compiled 1 file.
```

Comment:
I thought about putting this into `stderr` instead of `stdout` (I wouldn't need to adjust all tests, just add a new one), but based on experience in Jest CLI, I'd rather avoid that, as users tend to abuse this stream for catching their own errors or such.

Reply:
stdout is imho a perfect place for this, stderr implies some kind of error, warning at least, and this is a regular info message